cmd/bbolt/main_test.go

```diff
@@ -304,12 +303,6 @@ func NewMain() *Main {
 }
 
 func TestCompactCommand_Run(t *testing.T) {
-	var s int64
-	if err := binary.Read(crypto.Reader, binary.BigEndian, &s); err != nil {
-		t.Fatal(err)
-	}
-	rand.Seed(s)
```

Comment:
Please raise a PR to remove them from `main` as well, thanks.

Reply:
Done according to request!